model_2/db_supabase_connect.js

```diff
@@ -0,0 +1,40 @@
+const {createClient} =  "@supabase/supabase-js"
+const { Pool } = require('pg');
```

Comment:
Do we not need `require('dotenv').config();` to access env variables or has that changed?